driver-core/src/test/resources/change-streams/README.rst

```diff
@@ -0,0 +1,157 @@
+.. role:: javascript(code)
```

Comment:
I dont think we need this file.

Reply:
I started adding them in because it was good documentation on how to interpret the json files without having to go to the specs directory for it.  And if the README changes, the driver has a copy that reflects the version of the tests that the driver has.

Do you mind if we keep it?